.babelrc

```diff
@@ -0,0 +1,3 @@
+{
+  "presets": ["@babel/preset-env"]
```

Comment:
is there a way to avoid adding so many config files to our root directory?

Reply:
Idk, it is a key dependency to let us use ES6 module in jest